lib/ClangImporter/ClangDerivedConformances.cpp

```diff
@@ -394,6 +394,24 @@ bool swift::isIterator(const clang::CXXRecordDecl *clangDecl) {
   return getIteratorCategoryDecl(clangDecl);
 }
 
+ValueDecl *
+swift::importer::getImportedMemberOperator(const DeclBaseName &name,
+                                           NominalTypeDecl *selfType,
+                                           std::optional<Type> parameterType) {
+  assert(name.isOperator());
+  if (name.getIdentifier() == selfType->getASTContext().Id_EqualsOperator) {
+    return getEqualEqualOperator(selfType);
+  }
+  if (name.getIdentifier() == selfType->getASTContext().getIdentifier("-")) {
+    return getMinusOperator(selfType);
+  }
+  if (name.getIdentifier() == selfType->getASTContext().getIdentifier("+=") &&
+      parameterType) {
+    return getPlusEqualOperator(selfType, *parameterType);
+  }
+  return nullptr;
```

Comment:
Nit: unnecessary braces.

Why explicitly `==`, `-`, and `+=`? What about the other named operators?

Reply:
We need specifically these ones, as clang importer has special logic to instantiate & synthesize these operators when conforming C++ iterator types to the CxxIterator protocol .

Reply:
Added a comment.